docs/tutorials/quick_start.md

```diff
@@ -29,8 +29,10 @@ To quickly run a simple training example locally:
 
 ```
 cd example/mnist
```

Comment:
Thanks for your fix. https://github.com/bytedance/fedlearner/blob/master/example/sparse_model/test.sh here is a script we use to run a quick demo, easier than running 2 terminals. Can you add a script and update this doc correspondingly?

Reply:
OK,  I will  update

Reply:
@duanbing done